React/Modules/RCTDevMenu.m

```diff
@@ -185,7 +185,7 @@ - (instancetype)init
       [weakSelf.bridge.eventDispatcher sendDeviceEventWithName:@"toggleElementInspector" body:nil];
     }]];
 
-    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"Chrome";
+    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"JS Remotely";
```

Comment:
Safari already supports this out of the box and gets better with each release, including this year's OS X update. However the React hierarchy debugger tool would need to be rewritten as a WebKit debugger extension (currently it runs only in Chrome or Firefox) and I'm not sure whether that is possible.